.github/workflows/e2e.yaml

```diff
@@ -64,90 +64,8 @@ jobs:
           minikube image load kmm_local.tar
           minikube image load kmm-kmod_local.tar
 
-      - name: Cache binaries needed by Makefile
-        id: cache-bin
-        uses: actions/cache@v3
-        with:
-          path: ./bin
-          key: ${{ runner.os }}-bin-${{ env.GO_VERSION }}-${{ hashFiles('Makefile') }}
-
-      - uses: actions/setup-go@v3
-        with:
-          go-version: ${{ env.GO_VERSION }}
-        if: steps.cache-bin.outputs.cache-hit != 'true'
```

Comment:
Shall we keep this?

Reply:
If we keep it should be extracted from the tests and move to the "infra" part that should run only once per PR (and not for each job).

Reply:
Every job runs in a different VM, so we cannot really run this once. Maybe both steps could be combined into a single reusable action, but that action would need to be called in each job that needs the binaries (all of them as of today).